climateconnect/climateconnect
# Fix: FAQ translations

## Before merging
Run migrations and go to django admin to select which section should be shown on the about page

## Description
solves issue #991 
Backend changes:
- to FAQ which updates the serializer to access the translations for the FAQs. 
- a new url to access a new FAQ view (AboutFaqView)
- new field added to FaqSection that identifies which section should be shown on the About page


Frontend changes include:
 - Filtering questions is now case insenstive
 - Previously unseen text is now properly displayed
 - A link is provided to send an email if a question isn't listed in the FAQ
 - In the German version of the FAQ, the background image properly loads
 - HowItWorks section added small left margin to text in the carousel
 - About page makes a different api call to the new view to get the FAQs by section

## TODO: 
- (On production) Question: How do i create an account? -> Should be: How do I create an account? 
- Live Database needs to be modified to use the new field in the FaqSection model (Basics section ticked to True)

## Test plan

You can see the FAQ changes by doing of the following:
1. Open the about page and scroll down to the "got a question section" and then change the language and repeat
2. Open the FAQ page and then change the language and repeat

Filtering Questions:
- Start typing the FAQ you want to search for

Email a Question:
- Press the link above the searchbar

## Before landing

1. PR has meaningful title
1. `yarn lint` passes (frontend)
1. `yarn format` passes (frontend)
1. `make format` passes (backend)
